docs/CONFIG-VARS.md

```diff
@@ -122,6 +122,7 @@ the existing resources:
 | Name | Description | Type | Default | Notes |
 | :--- | ---: | ---: | ---: | ---: |
 | resource_group_name | Name of pre-existing resource group | string | null | Only required if deploying into existing resource group. |
+| vnet_resource_group_name | Name of a pre-exising resource group containing the BYO vnet resource | string | null | Only required if deploying into existing vnet that is not located in the resource_group_name specified above |
```

Comment:
@bernardmaltais given the extra resource group, is there a need for vnet/rg peering information here? Wanting to make sure there are not connectivity issues if one uses the current rg and the added `vnet_resource_group` here.